File: src/main/java/macro/library/console/Reader.java
package macro.library.console;

import org.jetbrains.annotations.NotNull;

import java.util.Scanner;

/**
 * Created by Macro303 on 2019-Oct-21
 */
abstract class Reader {
	private static final Scanner READER = new Scanner(System.in);

	static String readConsole(@NotNull String text) {
		System.out.print(Colour.GREEN + text + " >> ");
		var input = READER.nextLine().trim();
		System.out.print(Colour.RESET);
		return input;
	}
}

File: src/main/kotlin/macro/library/console/Reader.kt
package macro.library.console

import java.util.*

/**
 * Created by Macro303 on 2019-Oct-30
 */
internal object Reader {
	private val READER = Scanner(System.`in`)

	fun readConsole(text: String): String {
		print("${Colour.GREEN}$text >> ")
		val input = READER.nextLine().trim()
		print(Colour.RESET)
		return input
	}
}